game/ui/city/scorescreen.cpp

```diff
@@ -10,18 +10,19 @@
 #include "game/state/city/base.h"
 #include "game/state/city/facility.h"
 #include "game/state/gamestate.h"
+#include "game/state/playerstatesnapshot.h"
 #include "game/state/shared/agent.h"
 #include "game/state/shared/organisation.h"
 #include "game/ui/base/recruitscreen.h"
 
 namespace OpenApoc
 {
-ScoreScreen::ScoreScreen(sp<GameState> state, bool showWeeklyUpkeep)
+ScoreScreen::ScoreScreen(PlayerStateSnapshot state, bool showWeeklyUpkeep)
```

Comment:
If this doesn't alter the snapshot, that can be const. Also it can be a reference, as theres no need for a copy here.